redwood-lazylayout-schema/src/main/kotlin/app/cash/redwood/lazylayout/widgets.kt

```diff
@@ -25,33 +25,175 @@ import app.cash.redwood.ui.Margin
 
 @Widget(1)
 public data class LazyList(
+  /**
+   * The layout orientation of the list.
```

Comment:
```suggestion
   * Whether the list should be vertically oriented.
```

Reply:
This line was copy-pasted from the [Compose UI docs](https://github.com/androidx/androidx/blob/540188236dea66e1565548ae50da4f6b121b4aba/compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/lazy/LazyList.kt#L60), but I agree that it isn't worded very well. Will use your suggestion!